TestFoundation/TestNSPropertyList.swift

```diff
@@ -90,4 +91,17 @@ class TestNSPropertyList : XCTestCase {
             XCTFail("value stored is not a string")
         }
     }
+    
+    func test_booleanProperty() {
+        let plistDocString = "<?xml version='1.0' encoding='utf-8'?><!DOCTYPE plist PUBLIC \"-//Apple//DTD PLIST 1.0//EN\" \"http://www.apple.com/DTDs/PropertyList-1.0.dtd\"> <plist version='1.0'><dict><key>state</key><true/></dict></plist>"
+        do {
+            let plistDoc = try XMLDocument(xmlString: plistDocString, options: [])
+            try plistDoc.validate()
+            let plist = try PropertyListSerialization.propertyList(from: plistDoc.xmlData, options: [], format: nil) as! [String: Any]
```

Comment:
Should use `as? [String: Any]` here to avoid interrupted execution due to force unwrapping of `nil`,